poetry.lock

```diff
@@ -1,530 +0,0 @@
-[[package]]
```

Comment:
Since we're deleting this, we should clean up pyproject.toml too, and make sure those deps make it into our requirements file. Can you take a look there too?

Reply:
This may be something a little different to deal with. Since the modules have their own requirements, I am not sure you'll want all the "dev" dependencies going up with them? It seems that everything in pyproject.toml is dev related. Let me know, and maybe I am not quite getting your point here.

Reply:
I did ignore it and remove it from the commits.